shared/tasks/package.json

```diff
@@ -33,5 +32,8 @@
     "slash": "^5.0.0",
     "terser": "^5.17.1",
     "yargs-parser": "^21.1.1"
+  },
+  "dependencies": {
```

Comment:
@romaricpascal Mind if we keep this as a `devDependency`?

You probably wanted `--save-dev` not `--save` when fixing

Otherwise all the workspaces that import `govuk-frontend-tasks` will install it

Reply:
Oh yeah, forgot to check which kind of dependency it was before running the command , sorry 🤦🏻 I'll fix that 😊